### ISSUE TITLE
Smaller then 1em tagline? + very close to the left edge.

### ISSUE THREAD

**Issue by paaljoachim** [wplemon/gridd#128]
I am testing 2.0 beta

Customizer.
Site Identity

![Screen Shot 2019-10-30 at 16 23 14](https://user-images.githubusercontent.com/5323259/67895836-99929300-fb31-11e9-81d7-1bb316d441be.png)

I notice the Site Tagline Font-Size minimum is 1em.
Sometimes one might want this even smaller. Perhaps minimum 0.5 em similar to the Copyright Area?

+

Btw Also notice the blue circle is a bit too close to the edge of the browser.
There is a padding of 12px try 15px and see how that is.
(In Chrome it gives just a little more space and that helps.)

**Comment by aristath** [comment]
Changed minimum to `0.5`
Regarding the 12px/15px padding I tried that when I was implementing the control... the problem is that it then feels unbalanced. The whole customizer UI uses 12px for paddings/margins on the left & right. If I make that 15px then the whole slider gets 6px narrower than all other elements.
I _could_ however make the blue circle a bit smaller... Would that look better to you? It's currently defined as `1em`, which on my screen makes it ~20px. We could easily make that 15px instead
